test/catch2/utils.h

```diff
@@ -19,4 +20,37 @@ namespace utils {
 void break_here(const char *file, const char *func, int line);
 void throw_if_non_zero(int result);
 void wiredtiger_cleanup(const std::string &db_home);
+
+class shared_library {
```

Comment:
I see this works very similarly to load_extension, what was the context behind adding it here?